Sources/ssg/main.swift

```diff
@@ -5,19 +5,26 @@ import ArgumentParser
 struct SSGCommand: ParsableCommand {
 
     @Option(name: .shortAndLong, help: "Path to alternative keys file (defaults to .env)")
-    var environmentPath: String?
+    var environmentPath: String = ".env"
 
     @Option(name: .shortAndLong, help: "Path to variable checksum file")
     var checksumPath: String?
 
+    @Option(
+        name: .customLong("prioritize-alternative-keys"),
+        help: "Allows for prioritization of Alternative Keys before the Runtime Keys. When omitted, `false` is assumed."
+    )
+    var prioritizeAlternativeKeys: Bool = false
```

Comment:
instead of `@Option` you can use `@Flag`, it maps to a bool that's true when the flag is provided on the cmd line, and faster otherwise.

Reply:
also wondering if call this `--ignore-environment` might be clearer - certainly shorter and could have a non-ambiguous short representation (`-i`).. but either one works for me.

Reply:
I'm not sure about `ignore-environment` name because it doesn't indicate which environment is being ignored. Is it the runtime or the alternative?

If you don't think that is an issue, I can change the name.

Reply:
nope, I'm not beholden to it.. what you have works just as well.

Reply:
I switched to `Flag`, I was unaware of that option. Thanks!